html/elements/base.json

```diff
@@ -0,0 +1,217 @@
+                  },
+                  "edge_mobile": {
+                    "version_added": true
+                  },
+                  "firefox": {
+                    "version_added": "1.0"
+                  },
+                  "firefox_android": {
+                    "version_added": "1.0"
+                  },
+                  "ie": {
+                    "version_added": true,
+                    "notes": "Before Internet Explorer 7, <base> could be positioned anywhere in the document and the nearest value of <base> was used."
+                  },
+                  "ie_mobile": {
+                    "version_added": true
+                  },
+                  "opera": {
+                    "version_added": true
+                  },
+                  "opera_android": {
+                    "version_added": true
+                  },
+                  "safari": {
+                    "version_added": true
+                  },
+                  "safari_ios": {
+                    "version_added": true
+                  }
+                },
+                "status": {
+                  "experimental": false,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              }
+            }
+          },
+          "href": {
+            "__compat": {
+              "basic_support": {
+                "support": {
+                  "webview_android": {
+                    "version_added": true
+                  },
+                  "chrome": {
+                    "version_added": true
+                  },
+                  "chrome_android": {
+                    "version_added": true
+                  },
+                  "edge": {
+                    "version_added": true
+                  },
+                  "edge_mobile": {
+                    "version_added": true
+                  },
+                  "firefox": {
+                    "version_added": "1.0"
+                  },
+                  "firefox_android": {
+                    "version_added": "1.0"
+                  },
+                  "ie": {
+                    "version_added": true
+                  },
+                  "ie_mobile": {
+                    "version_added": true
+                  },
+                  "opera": {
+                    "version_added": true
+                  },
+                  "opera_android": {
+                    "version_added": true
+                  },
+                  "safari": {
+                    "version_added": true
+                  },
+                  "safari_ios": {
+                    "version_added": true
+                  }
+                },
+                "status": {
+                  "experimental": false,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              },
+              "relative_url": {
+                "desc": "Support of relative URIs.",
```

Comment:
"Support of" is unnecessary. Let's keep this concise: "Relative URIs" and the key could be "relative_uris", too.